install/ansible/uninstall.sh

```diff
@@ -140,8 +144,13 @@ fi
 echo "Uninstalling Contiv"
 
 # Uninstall contiv & API Proxy
-echo '- include: uninstall_auth_proxy.yml' >$ansible_path/uninstall_plays.yml
-echo '- include: uninstall_contiv.yml' >>$ansible_path/uninstall_plays.yml
+if [ $uninstall_v2plugin == true ]; then
+	echo '- include: uninstall_auth_proxy.yml' >$ansible_path/uninstall_plays.yml
+	echo '- include: uninstall_v2plugin.yml' >$ansible_path/uninstall_plays.yml
```

Comment:
This line overwrites uninstall_plays.yml, but the equivalent line in the `else` branch appends it.  Is this correct?